src/components/clusterConfiguration/NetworkConfiguration.tsx

```diff
@@ -92,15 +92,15 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
           id="networkConfigurationTypeAdvanced"
           name="networkConfigurationType"
           value="advanced"
-          isChecked={type === 'advanced'}
-          onChange={() => setType('advanced')}
+          isChecked={isAdvanced}
+          onChange={() => setAdvanced(true)}
```

Comment:
don't we need to use the the value of the radio button instead of using true?

Reply:
You actually cannot uncheck the radio button.
See screenshot

![Screenshot from 2020-08-26 13-45-15](https://user-images.githubusercontent.com/2078045/91299961-c3910200-e7a2-11ea-85fa-986962e510bc.png)

If Advanced Radio is checked, you cannot click on it to uncheck it, you have to click on Basic Radio. 
I guess that also answers your other question. We have two radio buttons which are mutually exclusive. You can select only one.

Reply:
So lets say that I'm trying to add a cluster and clicking on Advanced and then want to switch back to basic mode. what are my options to do it?

Reply:
The only option is to click on Basic radio